homeassistant/components/somfy_mylink/cover.py

```diff
@@ -64,6 +64,7 @@ class SomfyShade(RestoreEntity, CoverEntity):
 
     _attr_should_poll = False
     _attr_assumed_state = True
+    _attr_is_closed = None
```

Comment:
I missed that this should not be a class constant. Set it in init since it is updated by the state updates.